packages/astro/components/ViewTransitions.astro

```diff
@@ -75,6 +84,36 @@ const { fallback = 'animate' } = Astro.props as Props;
 			});
 		});
 
+		if(document.querySelector('[name="astro-view-transitions-forms"]')) {
+			document.addEventListener('submit', (ev) => {
+				let el = ev.target as HTMLElement;
+				if (
+				el.tagName !== 'FORM' ||
+					isReloadEl(el)
+				) {
+					return;
+				}
+
+				const form = el as HTMLFormElement;
+				const formData = new FormData(form);
+				const params = new URLSearchParams(formData as any); // TS is wrong here
+				let action = form.action;
+				const init: RequestInit = {
+					method: form.method,
+					redirect: 'follow'
+				} satisfies RequestInit;
```

Comment:
For redirect, 'follow' is the default value, 'get' is the default method for requests. For forms with method "get" we do not need that object (only for "post)

Reply:
Ah yeah, I was confused remembering when we had attempted to handle redirects ourselves and thought I needed to set that.